web-client/src/presenter/computeds/Public/publicTrialSessionHelper.test.ts

```diff
@@ -272,7 +272,7 @@ describe('publicTrialSessionsHelper', () => {
       expect(trialSessionRows).toEqual([
         {
           formattedSessionWeekStartDate: 'November 23, 2020',
-          sessionWeekStartDate: '2020-11-23T05:00:00.000+00:00',
+          sessionWeekStartDate: '2020-11-23T00:00:00.000-05:00',
```

Comment:
Looks like somehow a test was failing due to new timezone handling in test.

Reply:
You've handled it correctly, at least as far as we could tell when making these changes (publicTrialSessionHelper is not in staging). `createDateAtStartOfWeekEST` now takes in an ISO date string in with _any_ zone offset, and returns an EST offset date in UTC

Reply:
(at midnight)